eventually/authentication/models.py

```diff
@@ -79,11 +111,11 @@ def create(first_name, middle_name, last_name, email, password):
                           password=password)
         try:
             user.save()
-        except CustomUser.OperationalError():
-            pass
-        return user
+            return user
+        except django.db.utils.IntegrityError:
+            print('ERR: A user with such an email already exists')
 
-    def myattr_to_dict(self):
+    def to_dict(self):
         """
         :return: manually-defined attributes of the object. Inherited attributes aren't shown.
         :example: {'id': 8, 'first_name': 'fn', 'middle_name': 'mn',
```

Comment:
maybe it is a good idea to write each dict's field from a new line using `|` char.

Reply:
Fixed.